.aws/src/config/index.ts

```diff
@@ -44,13 +44,15 @@ const prefect = {
       'BRAZE_API_KEY',
       'BRAZE_REST_ENDPOINT',
       'SNOWFLAKE_DATA_RETENTION_ROLE',
+      'SNOWFLAKE_DATA_RETENTION_WAREHOUSE',
       'SNOWFLAKE_DATA_RETENTION_DB',
       'SNOWFLAKE_DATA_RETENTION_SCHEMA',
       'SNOWFLAKE_SNOWPLOW_DB',
       'SNOWFLAKE_SNOWPLOW_SCHEMA',
       'SNOWFLAKE_RAWDATA_DB',
       'SNOWFLAKE_RAWDATA_FIREHOSE_SCHEMA',
-      'SNOWFLAKE_DATA_RETENTION_WAREHOUSE',
+      'SNOWFLAKE_SNAPSHOT_DB',
```

Comment:
FWIW I am very intersted in making more sense of our databases and probably having fewer of them, meaning we would need fewer environment variables.